cmd/jwt.go

```diff
@@ -127,7 +124,7 @@ func webRequestAuthenticate(req *http.Request) error {
 		return errAuthentication
 	}
 	if err = claims.Valid(); err != nil {
-		return err
+		return errAuthentication
```

Comment:
Changing this to errAuthentication can make it harder to understand why authentication failed. I think the code calling webRequestAuthenticate() should say "Authentication failed" + the error returned by this function. We can do that in another PR.

Reply:
The reason to do this to ensure that some special checks by the caller work such as in Upload(), Download() webHandlers. From what i reviewed we have a bug here so i am making this change. UI changes for browser require a different level of work on this one.